docs/run-platform-server/configuring.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: Configuring
+sidebar_position: 30
+---
+
+import { Alert } from "@site/src/components/Alert";
+import { CodeExample } from "@site/src/components/CodeExample";
+
+Pre-requisites:
+
+- [Installation](./installing.mdx) is completed on a host machine and results in two executables: `stellar-horizon` and `stellar-core`. Per the installation instructions, `stellar-core` should be included in your host OS' `$PATH`, as the configurations below result in Horizon searching for a `stellar-core` to execute.
+- [Prepare the Database if first time](#prepare-the-database-first-time)
+
+You are now ready to setup configuration parameters to perform three important, distinct roles:
+
+- **serving requests** via a regular web-based HTTP API,
+- **ingesting ledgers** from core nodes of a Stellar network to keep its world-view up to date, and
+- **transaction submission** for sending transactions to the Stellar network.
+
+To perform these roles, you can choose from one of two of the deployment modes that follow. Each has its own configuration parameters.
+
+## Single Instance Deployment
+
+You run `stellar-horizon` in a single o/s process and it will perform all three roles at same time.
+
+| flag               | envvar         | example                              |
+| ------------------ | -------------- | ------------------------------------ |
+| `--db-url`         | `DATABASE_URL` | postgres://localhost/horizon_testnet |
+| `--network`        | `NETWORK`      | pubnet                               |
+| `--tx-sub-enabled` | TX_SUB_ENABLED | true                                 |
+
+## Multiple Instance Deployment
+
+You run multiple instances of `stellar-horizon` each performs a subset of the roles.
+
+### Ingestion Role Instance
+
+You need to allocate one instance to perform ongoing ingestion to capture network activity. This will default to limiting storage of ingested network activity in the database to a sliding window of the last 30 days.
+
+** TODO - make --history-retention-count default to 518400 **
```

Comment:
yes, on second thought, will include recommendation for setting `--history-retention-count=518400` explicitly in docs, as the minor gain of not setting one parameter seems outweighed by the potential confusion that would be caused otherwise.